modules/social_features/social_event/social_event.install

```diff
@@ -2209,3 +2209,17 @@ function social_event_update_11901(): string {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Fix email sending when user enrolling to an event.
+ */
+function social_event_update_11902(): string {
```

Comment:
I wanted to propose to change update hook number too start with 118, but since we already release 11901 in the 11.8.x in the PR [3328](https://github.com/goalgorilla/open_social/pull/3382/files), we should keep it like it is.